module-2/lab-matplotlib-seaborn/your-code/challenge-1.ipynb

```diff
@@ -90,14 +90,24 @@
   },
```

Comment:
- Why didn't you finish this one? Is there any question?
Plot both y=x^2 and y=exp(x) in the same plot using normal and logarithmic scale.